plugin/src/init.server.lua

```diff
@@ -134,19 +135,18 @@ local function sendFullDMInfo(isSilent: boolean?)
 
 	if not success then
 		warn(`[Luau Language Server] Connecting to server failed: {result}`)
+		cleanup()
 		connected.Value = false
```

Comment:
Ah nice spot, we should be able to get rid of the .Value set since that's handled by cleanup
```suggestion
		cleanup()
```

Reply:
Whoops, looks like this suggestion wasn't applied. I'll do it manually